clitests/script.sh

```diff
@@ -60,22 +59,24 @@ echo -e "\n\n"
 
 
 # github
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/00100"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/10110"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/11110"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/11111"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/uu1uu"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p ignore_commented_badges"
+SLEEP_DURATION=20
```

Comment:
```suggestion
SLEEP_DURATION=0
```

Reply:
Agree, result will not be changed by sleeping anyway